CreateZPTTask OdinZPTCreateZPT (lambda)

# 🐛 Description

 "replacement dimension 'era5_gmh' is not a 1D variable along the old dimension 'level'",

# 🔢 To reproduce

for input
``` json
{
  "name": "ac1/350/35021df9.ac1",
  "type": "ac1",
  "ScansInfo": [
    {
      "StatusCode": 200,
      "ScansInfo": [
        {
          "AltStart": 105976.18,
          "AltEnd": 72272.016,
          "LatStart": -80.33139,
          "LatEnd": -74.67873,
          "LonStart": 269.46375,
          "LonEnd": 166.97285,
          "MJDStart": 60213.3810633,
          "MJDEnd": 60213.3845215,
          "ScanID": 14229309538,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73863.64,
          "AltEnd": 9145.85,
          "LatStart": -74.491196,
          "LatEnd": -68.20576,
          "LonStart": 166.56259,
          "LonEnd": 157.16237,
          "MJDStart": 60213.3845677,
          "MJDEnd": 60213.3855796,
          "ScanID": 14229314383,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8233.713,
          "AltEnd": 71183.484,
          "LatStart": -68.07057,
          "LatEnd": -64.31486,
          "LonStart": 157.019,
          "LonEnd": 153.47957,
          "MJDStart": 60213.3856028,
          "MJDEnd": 60213.3865915,
          "ScanID": 14229315806,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73790.02,
          "AltEnd": 8956.513,
          "LatStart": -64.13202,
          "LatEnd": -57.436832,
          "LonStart": 153.33167,
          "LonEnd": 149.13074,
          "MJDStart": 60213.3866378,
          "MJDEnd": 60213.3876613,
          "ScanID": 14229317261,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 7883.514,
          "AltEnd": 70997.45,
          "LatStart": -57.293365,
          "LatEnd": -53.447018,
          "LonStart": 149.0564,
          "LonEnd": 147.11024,
          "MJDStart": 60213.3876844,
          "MJDEnd": 60213.3886732,
          "ScanID": 14229318684,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73694.46,
          "AltEnd": 9732.74,
          "LatStart": -53.262886,
          "LatEnd": -46.482544,
          "LonStart": 147.02522,
          "LonEnd": 144.48125,
          "MJDStart": 60213.3887195,
          "MJDEnd": 60213.389743,
          "ScanID": 14229320139,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8476.799,
          "AltEnd": 68447.75,
          "LatStart": -46.333694,
          "LatEnd": -42.573696,
          "LonStart": 144.4313,
          "LonEnd": 143.11186,
          "MJDStart": 60213.3897661,
          "MJDEnd": 60213.3907202,
          "ScanID": 14229321562,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73730.09,
          "AltEnd": 9974.448,
          "LatStart": -42.20068,
          "LatEnd": -35.437904,
          "LonStart": 142.98868,
          "LonEnd": 141.15244,
          "MJDStart": 60213.3908127,
          "MJDEnd": 60213.3918246,
          "ScanID": 14229323033,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8576.419,
          "AltEnd": 67857.98,
          "LatStart": -35.285187,
          "LatEnd": -31.497858,
          "LonStart": 141.1139,
          "LonEnd": 140.05568,
          "MJDStart": 60213.3918478,
          "MJDEnd": 60213.3928018,
          "ScanID": 14229324440,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73335.33,
          "AltEnd": 8568.225,
          "LatStart": -31.128119,
          "LatEnd": -24.063635,
          "LonStart": 139.95583,
          "LonEnd": 138.392,
          "MJDStart": 60213.3928944,
          "MJDEnd": 60213.3939526,
          "ScanID": 14229325911,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 9035.644,
          "AltEnd": 70912.71,
          "LatStart": -24.0118,
          "LatEnd": -20.151806,
          "LonStart": 138.37968,
          "LonEnd": 137.43872,
          "MJDStart": 60213.3939642,
          "MJDEnd": 60213.3949414,
          "ScanID": 14229327366,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 73236.336,
          "AltEnd": 9318.426,
          "LatStart": -19.957315,
          "LatEnd": -13.066442,
          "LonStart": 137.39258,
          "LonEnd": 136.04161,
          "MJDStart": 60213.3949876,
          "MJDEnd": 60213.3960111,
          "ScanID": 14229328805,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8552.476,
          "AltEnd": 70263.95,
          "LatStart": -12.925973,
          "LatEnd": -8.991485,
          "LonStart": 136.01385,
          "LonEnd": 135.12271,
          "MJDStart": 60213.3960343,
          "MJDEnd": 60213.397023,
          "ScanID": 14229330228,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72793.91,
          "AltEnd": 9600.464,
          "LatStart": -8.8010645,
          "LatEnd": -1.8988552,
          "LonStart": 135.08096,
          "LonEnd": 133.80017,
          "MJDStart": 60213.3970693,
          "MJDEnd": 60213.3980928,
          "ScanID": 14229331683,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8591.872,
          "AltEnd": 69708.09,
          "LatStart": -1.7526549,
          "LatEnd": 2.2046006,
          "LonStart": 133.77281,
          "LonEnd": 132.89902,
          "MJDStart": 60213.3981159,
          "MJDEnd": 60213.3991047,
          "ScanID": 14229333106,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72375.34,
          "AltEnd": 8911.895,
          "LatStart": 2.392195,
          "LatEnd": 9.386065,
          "LonStart": 132.85776,
          "LonEnd": 131.56021,
          "MJDStart": 60213.399151,
          "MJDEnd": 60213.400186,
          "ScanID": 14229334561,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8374.105,
          "AltEnd": 69755.72,
          "LatStart": 9.522459,
          "LatEnd": 13.482232,
          "LonStart": 131.53348,
          "LonEnd": 130.63635,
          "MJDStart": 60213.4002092,
          "MJDEnd": 60213.4011979,
          "ScanID": 14229336000,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72177.14,
          "AltEnd": 9238.92,
          "LatStart": 13.675859,
          "LatEnd": 20.61211,
          "LonStart": 130.5929,
          "LonEnd": 129.22836,
          "MJDStart": 60213.4012442,
          "MJDEnd": 60213.4022677,
          "ScanID": 14229337455,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8361.03,
          "AltEnd": 69214.63,
          "LatStart": 20.756096,
          "LatEnd": 24.733273,
          "LonStart": 129.19826,
          "LonEnd": 128.22647,
          "MJDStart": 60213.4022908,
          "MJDEnd": 60213.4032796,
          "ScanID": 14229338878,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71810.91,
          "AltEnd": 9560.645,
          "LatStart": 24.92381,
          "LatEnd": 31.849108,
          "LonStart": 128.17938,
          "LonEnd": 126.63573,
          "MJDStart": 60213.4033259,
          "MJDEnd": 60213.4043494,
          "ScanID": 14229340333,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8534.867,
          "AltEnd": 66914.26,
          "LatStart": 31.996454,
          "LatEnd": 35.842743,
          "LonStart": 126.60064,
          "LonEnd": 125.52576,
          "MJDStart": 60213.4043725,
          "MJDEnd": 60213.4053265,
          "ScanID": 14229341756,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71966.13,
          "AltEnd": 9867.467,
          "LatStart": 36.225906,
          "LatEnd": 43.071144,
          "LonStart": 125.415115,
          "LonEnd": 123.52253,
          "MJDStart": 60213.4054191,
          "MJDEnd": 60213.406431,
          "ScanID": 14229343227,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8612.45,
          "AltEnd": 66769.45,
          "LatStart": 43.222946,
          "LatEnd": 47.060707,
          "LonStart": 123.47653,
          "LonEnd": 122.112755,
          "MJDStart": 60213.4064542,
          "MJDEnd": 60213.4074082,
          "ScanID": 14229344634,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72012.84,
          "AltEnd": 10114.815,
          "LatStart": 47.43772,
          "LatEnd": 54.23178,
          "LonStart": 121.97028,
          "LonEnd": 119.33436,
          "MJDStart": 60213.4075008,
          "MJDEnd": 60213.4085127,
          "ScanID": 14229346105,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8725.939,
          "AltEnd": 66623.74,
          "LatStart": 54.385086,
          "LatEnd": 58.18858,
          "LonStart": 119.26625,
          "LonEnd": 117.26922,
          "MJDStart": 60213.4085358,
          "MJDEnd": 60213.4094899,
          "ScanID": 14229347512,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71992.016,
          "AltEnd": 8469.063,
          "LatStart": 58.55817,
          "LatEnd": 65.49853,
          "LonStart": 117.05622,
          "LonEnd": 112.34729,
          "MJDStart": 60213.4095824,
          "MJDEnd": 60213.4106406,
          "ScanID": 14229348983,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8757.613,
          "AltEnd": 69914.75,
          "LatStart": 65.55261,
          "LatEnd": 69.278885,
          "LonStart": 112.29988,
          "LonEnd": 108.438095,
          "MJDStart": 60213.4106522,
          "MJDEnd": 60213.4116294,
          "ScanID": 14229350438,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72224.56,
          "AltEnd": 9289.925,
          "LatStart": 69.46412,
          "LatEnd": 75.75317,
          "LonStart": 108.21363,
          "LonEnd": 97.26566,
          "MJDStart": 60213.4116757,
          "MJDEnd": 60213.4126992,
          "ScanID": 14229351877,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8446.1455,
          "AltEnd": 69837.89,
          "LatStart": 75.87611,
          "LatEnd": 79.06983,
          "LonStart": 96.94453,
          "LonEnd": 85.26899,
          "MJDStart": 60213.4127223,
          "MJDEnd": 60213.4137111,
          "ScanID": 14229353300,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72326.805,
          "AltEnd": 9588.477,
          "LatStart": 79.21098,
          "LatEnd": 82.49899,
          "LonStart": 84.54086,
          "LonEnd": 43.849186,
          "MJDStart": 60213.4137574,
          "MJDEnd": 60213.4147808,
          "ScanID": 14229354755,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8515.223,
          "AltEnd": 69625.6,
          "LatStart": 82.51379,
          "LatEnd": 81.83337,
          "LonStart": 42.698895,
          "LonEnd": 12.954204,
          "MJDStart": 60213.414804,
          "MJDEnd": 60213.4157927,
          "ScanID": 14229356178,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72291.49,
          "AltEnd": 8599.586,
          "LatStart": 81.75411,
          "LatEnd": 77.010666,
          "LonStart": 11.726344,
          "LonEnd": 341.3467,
          "MJDStart": 60213.415839,
          "MJDEnd": 60213.4168856,
          "ScanID": 14229357633,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8446.652,
          "AltEnd": 67244.484,
          "LatStart": 76.95641,
          "LatEnd": 73.69513,
          "LonStart": 341.17282,
          "LonEnd": 333.0216,
          "MJDStart": 60213.4168972,
          "MJDEnd": 60213.4178397,
          "ScanID": 14229359072,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72404.86,
          "AltEnd": 9039.996,
          "LatStart": 73.35469,
          "LatEnd": 66.849144,
          "LonStart": 332.36487,
          "LonEnd": 323.85928,
          "MJDStart": 60213.4179323,
          "MJDEnd": 60213.4189673,
          "ScanID": 14229360527,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8424.269,
          "AltEnd": 9270.173,
          "LatStart": 66.77748,
          "LatEnd": 55.96464,
          "LonStart": 323.7918,
          "LonEnd": 316.4594,
          "MJDStart": 60213.4189789,
          "MJDEnd": 60213.4210504,
          "ScanID": 14229361950,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8595.485,
          "AltEnd": 66638.1,
          "LatStart": 55.889065,
          "LatEnd": 52.16779,
          "LonStart": 316.42325,
          "LonEnd": 314.64578,
          "MJDStart": 60213.421062,
          "MJDEnd": 60213.4220045,
          "ScanID": 14229364830,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72076.47,
          "AltEnd": 9497.848,
          "LatStart": 51.799473,
          "LatEnd": 44.89009,
          "LonStart": 314.4837,
          "LonEnd": 312.02985,
          "MJDStart": 60213.422097,
          "MJDEnd": 60213.4231321,
          "ScanID": 14229366285,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8632.969,
          "AltEnd": 69549.086,
          "LatStart": 44.809483,
          "LatEnd": 40.809483,
          "LonStart": 312.00464,
          "LonEnd": 310.66202,
          "MJDStart": 60213.4231437,
          "MJDEnd": 60213.424144,
          "ScanID": 14229367708,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71946.875,
          "AltEnd": 9720.872,
          "LatStart": 40.61653,
          "LatEnd": 33.722866,
          "LonStart": 310.60114,
          "LonEnd": 308.80008,
          "MJDStart": 60213.4241903,
          "MJDEnd": 60213.4252138,
          "ScanID": 14229369179,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8791.606,
          "AltEnd": 68981.36,
          "LatStart": 33.64034,
          "LatEnd": 29.601395,
          "LonStart": 308.78015,
          "LonEnd": 307.68817,
          "MJDStart": 60213.4252253,
          "MJDEnd": 60213.4262257,
          "ScanID": 14229370586,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71555.086,
          "AltEnd": 8450.1,
          "LatStart": 29.411493,
          "LatEnd": 22.33613,
          "LonStart": 307.6385,
          "LonEnd": 306.11362,
          "MJDStart": 60213.4262719,
          "MJDEnd": 60213.4273186,
          "ScanID": 14229372057,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8428.788,
          "AltEnd": 66815.77,
          "LatStart": 22.21042,
          "LatEnd": 18.486496,
          "LonStart": 306.08618,
          "LonEnd": 305.19534,
          "MJDStart": 60213.4273417,
          "MJDEnd": 60213.4282726,
          "ScanID": 14229373512,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71252.08,
          "AltEnd": 8590.211,
          "LatStart": 18.0883,
          "LatEnd": 11.054855,
          "LonStart": 305.10394,
          "LonEnd": 303.74713,
          "MJDStart": 60213.4283652,
          "MJDEnd": 60213.4294002,
          "ScanID": 14229374951,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8346.768,
          "AltEnd": 71201.555,
          "LatStart": 10.92374,
          "LatEnd": 6.7844715,
          "LonStart": 303.72095,
          "LonEnd": 302.79602,
          "MJDStart": 60213.4294234,
          "MJDEnd": 60213.4304469,
          "ScanID": 14229376390,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 66373.15,
          "AltEnd": 8738.701,
          "LatStart": 6.167986,
          "LatEnd": -0.26839557,
          "LonStart": 302.68076,
          "LonEnd": 301.49634,
          "MJDStart": 60213.4305394,
          "MJDEnd": 60213.4314819,
          "ScanID": 14229377957,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8356.911,
          "AltEnd": 66143.27,
          "LatStart": -0.33988464,
          "LatEnd": -4.1725197,
          "LonStart": 301.4829,
          "LonEnd": 300.64206,
          "MJDStart": 60213.4314935,
          "MJDEnd": 60213.4324359,
          "ScanID": 14229379252,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71020.7,
          "AltEnd": 9008.096,
          "LatStart": -4.5637145,
          "LatEnd": -11.62153,
          "LonStart": 300.5571,
          "LonEnd": 299.24127,
          "MJDStart": 60213.4325285,
          "MJDEnd": 60213.4335636,
          "ScanID": 14229380707,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8312.556,
          "AltEnd": 68777.86,
          "LatStart": -11.763271,
          "LatEnd": -15.79931,
          "LonStart": 299.21353,
          "LonEnd": 298.2992,
          "MJDStart": 60213.4335867,
          "MJDEnd": 60213.4345755,
          "ScanID": 14229382146,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71100.04,
          "AltEnd": 9150.093,
          "LatStart": -15.998318,
          "LatEnd": -22.992937,
          "LonStart": 298.25455,
          "LonEnd": 296.85254,
          "MJDStart": 60213.4346218,
          "MJDEnd": 60213.4356452,
          "ScanID": 14229383601,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8291.577,
          "AltEnd": 68710.58,
          "LatStart": -23.138353,
          "LatEnd": -27.183466,
          "LonStart": 296.8215,
          "LonEnd": 295.81592,
          "MJDStart": 60213.4356684,
          "MJDEnd": 60213.4366571,
          "ScanID": 14229385024,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 71259.66,
          "AltEnd": 8805.248,
          "LatStart": -27.377277,
          "LatEnd": -34.369865,
          "LonStart": 295.76724,
          "LonEnd": 294.14957,
          "MJDStart": 60213.4367034,
          "MJDEnd": 60213.4377269,
          "ScanID": 14229386479,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 7288.9673,
          "AltEnd": 69146.84,
          "LatStart": -34.528996,
          "LatEnd": -38.537956,
          "LonStart": 294.1104,
          "LonEnd": 292.93637,
          "MJDStart": 60213.43775,
          "MJDEnd": 60213.4387388,
          "ScanID": 14229387902,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72148.836,
          "AltEnd": 9555.1045,
          "LatStart": -38.721046,
          "LatEnd": -45.680172,
          "LonStart": 292.88007,
          "LonEnd": 290.8408,
          "MJDStart": 60213.4387851,
          "MJDEnd": 60213.4398086,
          "ScanID": 14229389357,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8299.316,
          "AltEnd": 66990.016,
          "LatStart": -45.832592,
          "LatEnd": -49.703136,
          "LonStart": 290.7906,
          "LonEnd": 289.30746,
          "MJDStart": 60213.4398317,
          "MJDEnd": 60213.4407857,
          "ScanID": 14229390780,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72741.67,
          "AltEnd": 9882.769,
          "LatStart": -50.072872,
          "LatEnd": -56.89402,
          "LonStart": 289.15527,
          "LonEnd": 286.20956,
          "MJDStart": 60213.4408783,
          "MJDEnd": 60213.4418902,
          "ScanID": 14229392251,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 8531.15,
          "AltEnd": 67414.414,
          "LatStart": -57.046337,
          "LatEnd": -60.856113,
          "LonStart": 286.13275,
          "LonEnd": 283.85925,
          "MJDStart": 60213.4419134,
          "MJDEnd": 60213.4428674,
          "ScanID": 14229393658,
          "FreqMode": 2,
          "Backend": "AC1"
        },
        {
          "AltStart": 72402.41,
          "AltEnd": 8282.354,
          "LatStart": -61.236485,
          "LatEnd": -68.1329,
          "LonStart": 283.60806,
          "LonEnd": 277.992,
          "MJDStart": 60213.44296,
          "MJDEnd": 60213.4440182,
          "ScanID": 14229395129,
          "FreqMode": 2,
          "Backend": "AC1"
        }
      ]
    },
    {
      "StatusCode": 200,
      "ScansInfo": [
        {
          "AltStart": 106687.82,
          "AltEnd": 10282.122,
          "LatStart": -47.047695,
          "LatEnd": -57.800365,
          "LonStart": 313.97696,
          "LonEnd": 309.48645,
          "MJDStart": 60213.3744697,
          "MJDEnd": 60213.376066,
          "ScanID": 14229300422,
          "FreqMode": 21,
          "Backend": "AC1"
        },
        {
          "AltStart": 8038.37,
          "AltEnd": 105191.97,
          "LatStart": -58.094795,
          "LatEnd": -64.3017,
          "LonStart": 309.3302,
          "LonEnd": 305.11603,
          "MJDStart": 60213.3761123,
          "MJDEnd": 60213.3776855,
          "ScanID": 14229302693,
          "FreqMode": 21,
          "Backend": "AC1"
        },
        {
          "AltStart": 107740.87,
          "AltEnd": 9059.417,
          "LatStart": -64.48655,
          "LatEnd": -74.74068,
          "LonStart": 304.96515,
          "LonEnd": 291.6267,
          "MJDStart": 60213.3777318,
          "MJDEnd": 60213.3793513,
          "ScanID": 14229304932,
          "FreqMode": 21,
          "Backend": "AC1"
        },
        {
          "AltStart": 8126.689,
          "AltEnd": 108797.68,
          "LatStart": -74.86834,
          "LatEnd": -80.12564,
          "LonStart": 291.34222,
          "LonEnd": 270.92026,
          "MJDStart": 60213.3793744,
          "MJDEnd": 60213.381017,
          "ScanID": 14229307203,
          "FreqMode": 21,
          "Backend": "AC1"
        },
        {
          "AltStart": 105976.18,
          "AltEnd": 64455.598,
          "LatStart": -80.33139,
          "LatEnd": -82.37366,
          "LonStart": 269.46375,
          "LonEnd": 217.10196,
          "MJDStart": 60213.3810633,
          "MJDEnd": 60213.3823125,
          "ScanID": 14229309538,
          "FreqMode": 21,
          "Backend": "AC1"
        }
      ]
    }
  ],
}
```

# ✌🏽 Expected behaviour

What should have happened

# 📷 Screenshots

If applicable

# 💻 Traceback

```
[ERROR] ValueError: replacement dimension 'era5_gmh' is not a 1D variable along the old dimension 'level'
Traceback (most recent call last):
  File "/var/task/handler/create_zpt_handler.py", line 104, in handler
    profiles = donaletty.makeprofile(scans)
  File "/var/task/handler/newdonalettyERANC.py", line 63, in makeprofile
    scan_on_interp_gmh = scans.groupby("ScanID").map(
  File "/var/task/xarray/core/groupby.py", line 1558, in map
    return self._combine(applied)
  File "/var/task/xarray/core/groupby.py", line 1580, in _combine
    combined = concat(applied, dim)
  File "/var/task/xarray/core/concat.py", line 252, in concat
    return _dataset_concat(
  File "/var/task/xarray/core/concat.py", line 468, in _dataset_concat
    datasets = list(datasets)
  File "/var/task/xarray/core/groupby.py", line 1557, in <genexpr>
    applied = (func(ds, *args, **kwargs) for ds in self._iter_grouped())
  File "/var/task/handler/newdonalettyERANC.py", line 64, in <lambda>
    lambda ds: ds.squeeze().swap_dims(level="era5_gmh").interp(
  File "/var/task/xarray/core/dataset.py", line 4396, in swap_dims
    raise ValueError(


```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -22,6 +22,7 @@ jobs:
           env:
             AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
             AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            AWS_REGION: eu-north-1
           run: |
             tox
 

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -19,6 +19,9 @@ jobs:
             pip install tox
 
         - name: Run unit tests
+          env:
+            AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           run: |
             tox
 

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -22,7 +22,6 @@ jobs:
           env:
             AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
             AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-            AWS_REGION: eu-north-1
           run: |
             tox
 

--- a/create_zpt/handler/check_zpt_handler.py
+++ b/create_zpt/handler/check_zpt_handler.py
@@ -7,7 +7,6 @@ import boto3
 from botocore.exceptions import ClientError
 from .log_configuration import logconfig
 
-logconfig()
 
 ZPT_BUCKET = "odin-zpt"
 ZPT_PATTERN = "{backend}/{prefix}/{filename}.parquet"
@@ -38,6 +37,7 @@ def assert_zpt_exists(
 
 
 def handler(event: dict[str, Any], context: dict[str, Any]) -> dict[str, Any]:
+    logconfig()
     filename = os.path.split(event["name"])[-1]
     prefix = filename[0:3]
     backend = event["type"]

--- a/create_zpt/handler/create_zpt_handler.py
+++ b/create_zpt/handler/create_zpt_handler.py
@@ -28,7 +28,7 @@ def get_scan_data(scans_info: list[dict[str, Any]]) -> DataArray:
     for d in scans_info:
         data.extend(d["ScansInfo"])
 
-    df = DataFrame.from_records(data)
+    df = DataFrame.from_records(data).drop_duplicates()
     df["MJDMid"] = (df["MJDStart"] + df["MJDEnd"]) * 0.5
     df["LatMid"], df["LonMid"] = get_scan_geo_loc(
         df["LatStart"],

--- a/create_zpt/handler/create_zpt_handler.py
+++ b/create_zpt/handler/create_zpt_handler.py
@@ -18,7 +18,6 @@ from .geos import gmh
 from .scan_data_descriptions import parameter_desc
 from .log_configuration import logconfig
 
-logconfig()
 
 AWS_REGION = "eu-north-1"
 
@@ -58,6 +57,7 @@ def get_scan_data(scans_info: list[dict[str, Any]]) -> DataArray:
 
 
 def handler(event: dict[str, Any], context: dict[str, Any]) -> dict[str, Any]:
+    logconfig()
     scans = get_scan_data(event["ScansInfo"])
 
     dates: list[dt.date] = list(set(

--- a/create_zpt/requirements.txt
+++ b/create_zpt/requirements.txt
@@ -15,10 +15,10 @@ jmespath==1.0.1
 multidict==6.0.4
 nrlmsise00==0.1.2
 numcodecs==0.12.1
-numpy==1.26.1
+numpy==1.26.2
 packaging==23.2
-pandas==2.1.2
-pyarrow==14.0.0
+pandas==2.1.3
+pyarrow==14.0.1
 python-dateutil==2.8.2
 pytz==2023.3.post1
 pyyaml==6.0.1
@@ -27,7 +27,7 @@ scipy==1.11.3
 six==1.16.0
 tzdata==2023.3
 urllib3==2.0.7
-wrapt==1.15.0
+wrapt==1.16.0
 xarray==2023.10.1
 yarl==1.9.2
 zarr==2.16.1

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,8 +1,20 @@
+aiobotocore==2.7.0
+    # via s3fs
+aiohttp==3.8.6
+    # via
+    #   aiobotocore
+    #   s3fs
+aioitertools==0.11.0
+    # via aiobotocore
+aiosignal==1.3.1
+    # via aiohttp
 asciitree==0.3.3
     # via zarr
+async-timeout==4.0.3
+    # via aiohttp
 attrs==23.1.0
     # via
-    #   -r odincal/requirements.in
+    #   aiohttp
     #   cattrs
     #   jsii
 aws-cdk-asset-awscli-v1==2.2.201
@@ -11,25 +23,23 @@ aws-cdk-asset-kubectl-v20==2.1.2
     # via aws-cdk-lib
 aws-cdk-asset-node-proxy-agent-v6==2.0.1
     # via aws-cdk-lib
-aws-cdk-lib==2.104.0
+aws-cdk-lib==2.106.1
     # via -r requirements.in
 aws-psycopg2==1.3.8
     # via -r cache_tables/requirements.in
-black==23.10.1
+black==23.11.0
     # via -r requirements-dev.in
-boto3==1.28.78
-    # via
-    #   -r odincal/requirements.in
-    #   -r requirements.in
-boto3-stubs==1.28.78
+boto3==1.28.64
+    # via -r requirements.in
+boto3-stubs==1.28.84
     # via -r requirements-dev.in
-botocore==1.31.78
+botocore==1.31.64
     # via
     #   -r requirements.in
+    #   aiobotocore
     #   boto3
-    #   s3fs
     #   s3transfer
-botocore-stubs==1.31.78
+botocore-stubs==1.31.84
     # via boto3-stubs
 build==1.0.3
     # via pip-tools
@@ -42,7 +52,9 @@ certifi==2023.7.22
 chardet==5.2.0
     # via tox
 charset-normalizer==3.3.2
-    # via requests
+    # via
+    #   aiohttp
+    #   requests
 click==8.1.7
     # via
     #   black
@@ -63,13 +75,17 @@ filelock==3.13.1
     # via
     #   tox
     #   virtualenv
+frozenlist==1.4.0
+    # via
+    #   aiohttp
+    #   aiosignal
 fsspec==2023.10.0
     # via s3fs
-greenlet==3.0.1
-    # via sqlalchemy
 idna==3.4
-    # via requests
-importlib-resources==6.1.0
+    # via
+    #   requests
+    #   yarl
+importlib-resources==6.1.1
     # via jsii
 iniconfig==2.0.0
     # via pytest
@@ -86,9 +102,13 @@ jsii==1.91.0
     #   constructs
 mockito==1.4.0
     # via -r requirements-dev.in
-mypy==1.6.1
+multidict==6.0.4
+    # via
+    #   aiohttp
+    #   yarl
+mypy==1.7.0
     # via -r requirements-dev.in
-mypy-boto3-lambda==1.28.63
+mypy-boto3-lambda==1.28.83
     # via boto3-stubs
 mypy-boto3-s3==1.28.55
     # via boto3-stubs
@@ -102,9 +122,8 @@ nrlmsise00==0.1.2
     # via -r create_zpt/requirements.in
 numcodecs==0.12.1
     # via zarr
-numpy==1.26.1
+numpy==1.26.2
     # via
-    #   -r odincal/requirements.in
     #   nrlmsise00
     #   numcodecs
     #   pandas
@@ -120,10 +139,9 @@ packaging==23.2
     #   pytest
     #   tox
     #   xarray
-pandas==2.1.2
+pandas==2.1.3
     # via
     #   -r create_zpt/requirements.in
-    #   -r odincal/requirements.in
     #   xarray
 pathspec==0.11.2
     # via black
@@ -138,8 +156,6 @@ pluggy==1.3.0
     # via
     #   pytest
     #   tox
-psycopg2-binary==2.9.9
-    # via -r odincal/requirements.in
 publication==0.0.3
     # via
     #   aws-cdk-asset-awscli-v1
@@ -148,12 +164,8 @@ publication==0.0.3
     #   aws-cdk-lib
     #   constructs
     #   jsii
-pyarrow==14.0.0
+pyarrow==14.0.1
     # via -r create_zpt/requirements.in
-pygresql==6.0
-    # via -r odincal/requirements.in
-pyinotify==0.9.6
-    # via -r odincal/requirements.in
 pyproject-api==1.6.1
     # via tox
 pyproject-hooks==1.0.0
@@ -171,10 +183,9 @@ pyyaml==6.0.1
     # via
     #   -r cache_tables/requirements.in
     #   -r create_zpt/requirements.in
-    #   -r odincal/requirements.in
 requests==2.31.0
     # via -r cache_tables/requirements.in
-s3fs==0.4.2
+s3fs==2023.10.0
     # via -r create_zpt/requirements.in
 s3transfer==0.7.0
     # via boto3
@@ -182,8 +193,6 @@ scipy==1.11.3
     # via -r create_zpt/requirements.in
 six==1.16.0
     # via python-dateutil
-sqlalchemy==2.0.23
-    # via -r odincal/requirements.in
 tomli==2.0.1
     # via
     #   black
@@ -204,7 +213,7 @@ typeguard==2.13.3
     #   aws-cdk-lib
     #   constructs
     #   jsii
-types-awscrt==0.19.8
+types-awscrt==0.19.10
     # via botocore-stubs
 types-psycopg2==2.9.21.15
     # via -r requirements-dev.in
@@ -226,7 +235,6 @@ typing-extensions==4.8.0
     #   mypy-boto3-lambda
     #   mypy-boto3-s3
     #   mypy-boto3-stepfunctions
-    #   sqlalchemy
 tzdata==2023.3
     # via pandas
 urllib3==1.26.18
@@ -238,8 +246,12 @@ virtualenv==20.24.6
     # via tox
 wheel==0.41.3
     # via pip-tools
+wrapt==1.16.0
+    # via aiobotocore
 xarray==2023.10.1
     # via -r create_zpt/requirements.in
+yarl==1.9.2
+    # via aiohttp
 zarr==2.16.1
     # via -r create_zpt/requirements.in
 

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,13 +2,13 @@ attrs==23.1.0
 aws-cdk-asset-awscli-v1==2.2.201
 aws-cdk-asset-kubectl-v20==2.1.2
 aws-cdk-asset-node-proxy-agent-v6==2.0.1
-aws-cdk-lib==2.104.0
-boto3==1.28.78
-botocore==1.31.78
+aws-cdk-lib==2.106.1
+boto3==1.28.84
+botocore==1.31.84
 cattrs==23.1.2
 constructs==10.3.0
 exceptiongroup==1.1.3
-importlib-resources==6.1.0
+importlib-resources==6.1.1
 jmespath==1.0.1
 jsii==1.91.0
 publication==0.0.3

--- a/tests/test_create_zpt.py
+++ b/tests/test_create_zpt.py
@@ -1,0 +1,48 @@
+from create_zpt.handler.create_zpt_handler import get_scan_data
+
+# Dataset from a failed statemachine, is this a split scan?
+scans = [
+    {"StatusCode": 204, "ScansInfo": []},
+    {
+        "StatusCode": 200,
+        "ScansInfo": [
+            {
+                "AltStart": 106905.08,
+                "AltEnd": 9460.85,
+                "LatStart": 71.514114,
+                "LatEnd": 61.195534,
+                "LonStart": 182.94179,
+                "LonEnd": 173.09192,
+                "MJDStart": 60244.8251785,
+                "MJDEnd": 60244.8267748,
+                "ScanID": 14272782480,
+                "FreqMode": 21,
+                "Backend": "AC1",
+            },
+        ],
+    },
+    {"StatusCode": 204, "ScansInfo": []},
+    {
+        "StatusCode": 200,
+        "ScansInfo": [
+            {
+                "AltStart": 106905.08,
+                "AltEnd": 9460.85,
+                "LatStart": 71.514114,
+                "LatEnd": 61.195534,
+                "LonStart": 182.94179,
+                "LonEnd": 173.09192,
+                "MJDStart": 60244.8251785,
+                "MJDEnd": 60244.8267748,
+                "ScanID": 14272782480,
+                "FreqMode": 21,
+                "Backend": "AC1",
+            },
+        ],
+    },
+]
+
+
+def test_get_scan_data():
+    data = get_scan_data(scans)
+    assert (1,) == data.ScanID.shape

--- a/tox.ini
+++ b/tox.ini
@@ -3,11 +3,6 @@ envlist = py310,mypy,lint
 skipsdist = True
 
 [testenv:py310]
-pass_env =
-    AWS_ACCESS_KEY_ID
-    AWS_SECRET_ACCESS_KEY
-    AWS_PROFILE
-    AWS_REGION
 deps =
     -rrequirements-dev.txt
     pytest

--- a/tox.ini
+++ b/tox.ini
@@ -2,14 +2,18 @@
 envlist = py310,mypy,lint
 skipsdist = True
 
-# [testenv:py310]
-# deps =
-#     -rrequirements-dev.txt
-#     pytest
-# setenv =
-#     TZ = utc
-# commands =
-#     pytest tests {posargs}
+[testenv:py310]
+pass_env =
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_PROFILE
+deps =
+    -rrequirements-dev.txt
+    pytest
+setenv =
+    TZ = utc
+commands =
+    pytest tests {posargs}
 
 [testenv:mypy]
 basepython = python3.10
@@ -33,3 +37,4 @@ max_line_length = 88
 filterwarnings =
     once::DeprecationWarning
     once::PendingDeprecationWarning
+addopts = --ignore=cdk.out --ignore=odincal

--- a/tox.ini
+++ b/tox.ini
@@ -7,6 +7,7 @@ pass_env =
     AWS_ACCESS_KEY_ID
     AWS_SECRET_ACCESS_KEY
     AWS_PROFILE
+    AWS_REGION
 deps =
     -rrequirements-dev.txt
     pytest

